.github/workflows/ci.yml

```diff
@@ -35,3 +39,5 @@ jobs:
       run: scripts/xcode_select_current_version.sh
     - name: scripts/xcodebuild_wrapper.sh ${{ matrix.build_command }}
       run: scripts/xcodebuild_wrapper.sh ${{ matrix.build_command }}
+    - script: pod lib lint
```

Comment:
Can we have this in a .sh file in the scripts folder instead of having the command directly here?

Reply:
what's the advantage of having a .sh file for this one line?

Reply:
It is a good practice in case we want to use it in other workflows. (like **publishCocoaPod.sh**)

One of the scenarios I can think of now is: If we need to update it in any way (add more validation commands to the script or simply changing/adding parameters of **pod lib lint** it'll be a one file change.

Reply:
My understanding about current framework is that the `.sh` script is for local validation and `.yml` script is for cloud validation. In most cases commands are executed in `.sh` script and wrapper are invoked in `.yml` . This way if the cloud build fails then we can use `.sh` script to repo error locally.

Reply:
the yml script can still do multilines and params. but Kunal argument on .sh for local ci.sh makes sense. I can go create one.

Reply:
I understand the YAML has support for params and multi lines.

The point I'm trying to make is about reusability of these commands/parameters when and if they need to be used in other YAML workflows.